lexer/src/test/java/api/pds/nasa/gov/api_search_query_lexer/TestParsing.java

```diff
@@ -113,14 +123,220 @@ void testTemporalRange() {
+    SearchLexer lex = new SearchLexer(input);
+    CommonTokenStream tokens = new CommonTokenStream(lex);
+    SearchParser par = new SearchParser(tokens);
+    ParseTree tree = par.query();
+    ParseTreeWalker walker = new ParseTreeWalker();
+    walker.walk(this, tree);
+    
+    Assertions.assertNull(this.field);
+    Assertions.assertNotNull(this.strval);
+    Assertions.assertEquals("\".*apple\"", this.strval.getSymbol().getText());
+  }
+  @Test
+  void testParenStrvalExistence() {
+    String queryString  = "(\".*apple\" exists)";
+    CodePointCharStream input = CharStreams.fromString(queryString);
+    SearchLexer lex = new SearchLexer(input);
+    CommonTokenStream tokens = new CommonTokenStream(lex);
+    SearchParser par = new SearchParser(tokens);
+    ParseTree tree = par.query();
+    ParseTreeWalker walker = new ParseTreeWalker();
+    walker.walk(this, tree);
+    
+    Assertions.assertNull(this.field);
+    Assertions.assertNotNull(this.strval);
+    Assertions.assertEquals("\".*apple\"", this.strval.getSymbol().getText());
+  }
+  @Override
+  public void enterQuery(QueryContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void exitQuery(QueryContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void enterQueryTerm(QueryTermContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void exitQueryTerm(QueryTermContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void enterGroup(GroupContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void exitGroup(GroupContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void enterExpression(ExpressionContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void exitExpression(ExpressionContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void enterAndStatement(AndStatementContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void exitAndStatement(AndStatementContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void enterOrStatement(OrStatementContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void exitOrStatement(OrStatementContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void enterComparison(ComparisonContext ctx) {
+    this.field = ctx.FIELD();
+    this.number = ctx.NUMBER();
+    this.strval = ctx.STRINGVAL();
+  }
+
+  @Override
+  public void exitComparison(ComparisonContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void enterOperator(OperatorContext ctx) {
+    // TODO Auto-generated method stub
+
+  }
 
+  @Override
+  public void exitOperator(OperatorContext ctx) {
+    // TODO Auto-generated method stub
```

Comment:
Minor comment, but I think we could replace those TODO comments with "Nothing needs to be done here."